atlasdb-jdbc/src/main/resources/META-INF/services/io.dropwizard.jackson.Discoverable

```diff
@@ -0,0 +1 @@
+com.palantir.atlasdb.jdbc.config.JdbcDataSourceConfiguration
```

Comment:
These file is auto generated by the annotaion @AutoService(Discoverable.class), shouldnt we keep the annotation rather than adding files explicitly?